modules/AOD_Index/LuceneUtils.php

```diff
@@ -244,7 +244,7 @@ function rtf2text($filename)
                     $param = null;
 
                     // Start reading characters after the backslash.
-                    for ($k = $i + 1, $m = 0; $k < strlen($text); $k++, $m++) {
+                    for ($k = $i + 1, $m = 0, $kMax = strlen($text); $k < $kMax; $k++, $m++) {
```

Comment:
Does this loop having an `$m` value as well make any difference for the code here?

Reply:
Nope, the behaviour should be the same as before with $k and $m getting returned and incremented by one every loop.